### PR TITLE
Fix HTML validation errors in review app 

### PR DESCRIPTION
**Pull request by hannalaakso**
This fixes the following:
- A document must not include more than one visible `main` element.
- Element `div` not allowed as child of element `h2` in this context

### CODE REVIEW

File: app/views/full-page-examples/bank-holidays/index.njk [hunk 1]
```diff
@@ -50,9 +50,10 @@ notes: >-
       {% set englandAndWalesHTML %}
 
         {% set englandAndWalesPanelTitleHtml %}
-          <div class='govuk-!-font-size-36 govuk-!-margin-bottom-4'>
+          <span class='govuk-!-font-size-36'>
             The next bank holiday in England and Wales is
-          </div>
+          </span>
+          <br><br>
```
Comment (nickcolley) [inline app/views/full-page-examples/bank-holidays/index.njk]: Is there a way to do this without the <br>? do we have a `govuk-!-display-block`? Could make an `app-` version maybe.

Comment (36degrees) [inline app/views/full-page-examples/bank-holidays/index.njk]: Wondering if we could/should just put `display: block` on the span? Not pretty, but neither is random line breaks.

File: app/views/full-page-examples/confirm-delete/index.njk [hunk 2]
```diff
@@ -9,40 +9,38 @@
 {% set mainClasses = "govuk-main-wrapper--l" %}
 
 {% block content %}
-  <main id="content" role="main">
```
Comment (36degrees) [inline app/views/full-page-examples/confirm-delete/index.njk]: Good catch 👏

File: app/views/full-page-examples/bank-holidays/index.njk (hunk 1 above)
Comment (hannalaakso) [reply app/views/full-page-examples/bank-holidays/index.njk]: Yes a good idea 👍

**Pull request merged**